c.html

```diff
@@ -62,12 +62,12 @@
 	var validPassport = 0
 
 	// Regex
-	var regex4digits = new RegExp("([0-9]{4})$")
-	var regexHeight = new RegExp("([0-9][A-Za-z]{2})$")
-	var regexHexaColor = new RegExp("^#([A-Fa-f0-9]{6}|[A-Fa-f0-9]{3})$")
-	var regexLeadingZeros = new RegExp(/[0-9]{4,100}$/);
+	var regex4digits = new RegExp("^([0-9]{4})$")
+	var regexHeight = new RegExp("^([0-9]+[A-Za-z]{2})$")
```

Comment:
Here, I'm adding a `^` and adding a `+` after the digit, because otherwise only a single digit is matched.